### PR TITLE
Add a new MSP2_INAV_MISC2 message type

### PR DESCRIPTION
**Pull request by danarrib**
New message for MSPv2 protocol on INAV.
It allows a peer to fetch 
* On Time
* Flight Time
* Throttle %
* Auto Throttle flag

### CODE REVIEW

File: src/main/fc/fc_msp.c [hunk 1]
```diff
@@ -824,6 +824,25 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
...
+        sbufWriteU32(dst, getFlightTime()); // Flight time (seconds)
+
+        // Throttle
+        int16_t thr = 0;
+        const int minThrottle = getThrottleIdleValue();
+        bool autoThrottle = navigationIsControllingThrottle();
+        if(autoThrottle)
+            thr = (constrain(rcCommand[THROTTLE], PWM_RANGE_MIN, PWM_RANGE_MAX) - PWM_RANGE_MIN) * 100 / (PWM_RANGE_MAX - PWM_RANGE_MIN);
+        else
+            thr = (constrain(rcCommand[THROTTLE], PWM_RANGE_MIN, PWM_RANGE_MAX ) - minThrottle) * 100 / (motorConfig()->maxthrottle - minThrottle);
```
Comment (DzikuVx) [inline src/main/fc/fc_msp.c]: Can't we read it from the Mixer somehow? I do not like the idea of computing this again here in the MSP part

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/fc/fc_msp.c (hunk 1 above)
Comment (danarrib) [reply src/main/fc/fc_msp.c]: Ok, I moved the computing from OSD to Mixer and created a new variable to store the value to be used anywhere. Then I use the new variable on OSD and on the new MSP message.

File: src/main/flight/mixer.c [hunk 2]
```diff
@@ -529,6 +530,12 @@ void FAST_CODE mixTable(const float dT)
 
     throttleRange = throttleMax - throttleMin;
 
+    // Throttle Percent used by OSD and MSP
+    if(navigationIsControllingThrottle())
```
Comment (DzikuVx) [inline src/main/flight/mixer.c]: I was rather hoping we could get a function in Mixer to compute this and use when needed in other places

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: src/main/flight/mixer.c (hunk 2 above)
Comment (danarrib) [reply src/main/flight/mixer.c]: Ok, fixed as requested.

**Review by DzikuVx — APPROVED** [review]

**Pull request merged**